src/python/grpcio/commands.py

```diff
@@ -58,6 +63,117 @@ class CommandError(Exception):
   """Simple exception class for GRPC custom commands."""
 
 
+# TODO(atash): Remove this once PyPI has better Linux bdist support
+def _linux_target_egg_name(grpc_version):
+  """Get gRPC's custom made-up target names for Linux."""
+  assert sys.platform.startswith('linux')
+  version = '.'.join(map(str, sys.version_info[0:2]))
+  architecture = platform.machine()
```

Comment:
There's a way to do this through `pkg_resources`, which should be equivalent to how `setuptools` gets the base distribution name. Will go with that instead.